### PR TITLE
Send budget caps in dollar amounts

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @zeantsoi 

per https://adzerk.slack.com/?redir=%2Farchives%2Fadzerk-reddit%2Fp1450477132000044

### CODE REVIEW

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -323,10 +324,13 @@ def update_flight(link, campaign, az_campaign):
...
             d['Impressions'] = campaign.impressions + ADZERK_IMPRESSION_BUMP
         else:
+            total_budget_dollars = campaign.total_budget_pennies / 100.
+            # budget caps must be whole dollar amounts. round things up to prevent
+            # things from underdelivering.
             d.update({
                 'CapType': 4,
-                'DailyCapAmount': campaign.total_budget_pennies / campaign.ndays,
-                'LifetimeCapAmount': campaign.total_budget_pennies,
+                'DailyCapAmount': math.ceil(total_budget_dollars / campaign.ndays),
```
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: Need to cast these to an `int` type:

    int('DailyCapAmount': math.ceil(total_budget_dollars / campaign.ndays)),
    int('LifetimeCapAmount': math.ceil(total_budget_dollars)),

**Comment by zeantsoi** [comment]
:nail_care:

**Comment by dwick** [comment]
:haircut:

File: reddit_adzerk/adzerkpromote.py [hunk 2]
```diff
@@ -323,10 +324,13 @@ def update_flight(link, campaign, az_campaign):
...
             d['Impressions'] = campaign.impressions + ADZERK_IMPRESSION_BUMP
         else:
+            total_budget_dollars = campaign.total_budget_pennies / 100.
+            # budget caps must be whole dollar amounts. round things up to prevent
+            # things from underdelivering.
             d.update({
                 'CapType': 4,
-                'DailyCapAmount': campaign.total_budget_pennies / campaign.ndays,
-                'LifetimeCapAmount': campaign.total_budget_pennies,
+                'DailyCapAmount': int(math.ceil(total_budget_dollars / campaign.ndays)),
```
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: Nit: exceeds 80 characters. Maybe break into multi-line?

**Comment by zeantsoi** [comment]
One nit, but :fish:

**Pull request merged**